stout/flags.cc

```diff
@@ -72,6 +126,39 @@ void Parser::AddOrExit(
 
 ////////////////////////////////////////////////////////////////////////
 
+google::protobuf::Message* Parser::TryParseSubcommand(const std::string& arg) {
+  CHECK(cur_subcommand_message_);
+
+  if (const google::protobuf::FieldDescriptor* subcommand_field =
+          cur_subcommand_message_
+              .value()
+              ->GetDescriptor()
+              ->FindFieldByName(arg);
+      subcommand_field == nullptr) {
```

Comment:
Any reason not to do the standard thing here:
```suggestion
  const auto* subcommand_field =
      cur_subcommand_message_.value()->GetDescriptor()->FindFieldByName(arg);
  
  if (subcommand_field == nullptr) {
```